### ISSUE TITLE
Allow HTTP for Console

### ISSUE THREAD

**Issue by mjnagel** [PaloAltoNetworks/prisma-cloud-compute-operator#13]
### Is your feature request related to a problem?

Currently the default setup using ClusterIP service type creates a service with HTTPS management port and the comms port. When using an ingress method like Istio without a passthrough gateway the HTTP 8081 port needs to be exposed to have the service mesh handle HTTPS.

### Describe the solution you'd like

Add an additional "toggle" in the spec to enable the HTTP 8081 port to be exposed.

### Additional context

I reviewed the resource spec document to see if there was an existing option but did not see anything. If there is definitely point me to it and I can test it out. Thanks!

**Comment by wfg** [comment]
You're right--there's no option to do this today. Thanks for the feedback @mjnagel.

**Comment by pfox1969** [comment]
This leads into having the ability to modify the twistlock.cfg and the resulting yamls from ```twistcli console/defender export kubernetes``` prior to calling Console / Defender creation.
